articles/compliance/index.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Auth0 and General Data Protection Regulation
+description: How Auth0 complies with the EU's General Data Protection Regulation (GDPR)
+classes: topic-page
+---
+
+<div class="topic-page-header">
+  <div data-name="example" class="topic-page-badge"></div>
+  <h1>General Data Protection Regulation</h1>
+  <p>
```

Comment:
Add a short text here (within the `topic-page-header` div) and move all this text after the `topic-page-header` div.

Currently, it looks like this:

![image](https://user-images.githubusercontent.com/11715799/32183164-3ce3a04c-bda1-11e7-87f9-e534c29f55b1.png)